### ISSUE TITLE
Network._compute_correct could be done in numpy

### ISSUE THREAD

**Issue by dsblank** [Calysto/conx#47]
Would make this much faster for larger datasets.

**Comment by dsblank** [comment]
Or even better: do it in Keras.

**Comment by dsblank** [comment]
Or in keras, and could use the actual error functions. That would allow crossentropy and other loss functions to be handled correctly.
